src/plugins/auto_batch/src/compiled_model.cpp

```diff
@@ -83,6 +85,8 @@ CompiledModel::GetWorkerInferRequest() const {
                 {
                     std::unique_lock<std::mutex> lock(workerRequestPtr->_mutex);
                     status = workerRequestPtr->_cond.wait_for(lock, std::chrono::milliseconds(m_time_out));
+                    if ((status != std::cv_status::timeout) && (workerRequestPtr->_is_wakeup == false))
```

Comment:
I doubt this is useful in batch logic..._is_wakeup is set to true when the subrequested is fully collected and subrequests are done, but never reset to false, how do  you ensure no spurious wake up if second or third infer requests?